### DISCUSSION TITLE
Add a new information line in documentation

### DISCUSSION THREAD

**Discussion by oliverigor27** [devopness/devopness#897] [Ideas / Feature requests]
### Problem to be solved

In the [Application Deploy Docs](https://www.devopness.com/docs/applications/deploy-application/), it should have an extra line to informing the user who will use this feature that he must set the server before he apply the deploy.

### Why does this matter?

Current it assume that the user already set up the server before try to deploy its application or the user can think his server is automatically set up. Would be good give them an extra adivice about that.

### Describe the solution you'd like.

Just a single line which will tell the user he needs to set up his server before try to deploy.

**Comment by jfoliveira** [comment]
Hi @oliverigor27 
Could you please have a look at this pull request with improvements in the application deploy docs?
https://github.com/devopness/devopness/pull/914 

What do you think of these new instructions?
